_data/simple-icons.json

```
@@ -11629,6 +11629,11 @@
             "hex": "6EBE49",
             "source": "https://branding.seagate.com/productpage/3fc51aba-c35a-4eff-a833-a258b0440bd2"
         },
+        {
+            "title": "SearXNG",
+            "hex": "3050ff",
+            "source": "https://docs.searxng.org/_static/searxng-wordmark.svg"
```

```suggestion
            "source": "https://docs.searxng.org"
```

We can get away with just linking to the homepage for this one.